scripts/properties2json.js

```diff
@@ -1,4 +1,4 @@
-var config = require('../intl-config.js');
+var defaultConfig = require('../intl-config.js');
```

Comment:
All of the changes in this file have one goal.

Making things generic enough so we can plug in different configs.